src/templates/console.html

```diff
@@ -58,6 +58,8 @@
         let await_fut = namespace.get("await_fut");
         let pyconsole = namespace.get("pyconsole");
         let clear_console = namespace.get("clear_console");
+        const echo = (msg, ...opts) =>
+          term.echo(msg.replace("]]", "&rsqb;&rsqb;"), ...opts);
```

Comment:
Thanks a lot! Yes, clearly something needed escaping but I wasn't sure what.

So the opening double bracket doesn't need escaping?


Thanks for fixing the vars -> let as well.

Reply:
> So the opening double bracket doesn't need escaping?

Yes, that would be safer. If we escape opening brackets too, we need to stop using templates in our console.html.

https://github.com/pyodide/pyodide/blob/379388c20c8cb43e6d0a9c796f7247c4879dce5b/src/templates/console.html#L102-L104

We are using it to print a colored truncation message.